static/api/v2.yaml

```diff
@@ -1749,79 +2468,59 @@ paths:
               examples:
                 Internal Error:
                   $ref: '#/paths/~1v2~1users/get/responses/500/content/application~1json/examples/Internal%20Error'
-  /v2/binding-jobs:
-    post:
+    put:
       tags:
-        - BindingJobs
-      operationId: CreateBindingJob
-      summary: Create a new credential binding job
+        - Passkeys
+      operationId: SetPasskeyTags
+      summary: List of tags to associated with a passkey. This list will replace any existing tags associated with a passkey. If a provided tag does not include an id, a new tag will be created.
       description: |
-        To create a binding job, send a POST request to `/v2/binding-jobs`. Values in the
-        request body for read-only attributes get ignored.
+        write a description about passkeys
```

Comment:
This should be something like "Sets the tags associated with a passkey. Any tags that don't exist will be created." 

We need to fully verify the behavior for this, there might be more to it. But IIRC this will either associated a tag with a UUID if it exists OR if will create a new tag with a new UUID.